extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -48,6 +49,20 @@ export let Message;
 export let RequestHandler;
 
 /**
+  * @param {*} message
+  * @return {?Message}
+  */
+export function parseMessage(message, isWebview_) {
```

Comment:
1. No underscore in the param
2. Let's also add it to `@param` in jsdoc above
3. We can do one of the two things:
a. Either rename `isWebview_` to something like `parseJson`.
b. Or simply infer it in the code by doing `typeof message == 'string'`